plugins/coverimage.koplugin/main.lua

```diff
@@ -0,0 +1,236 @@
+local Device = require("device")
+
+if not Device.isAndroid() and not Device.isEmulator() then
+    return { disabled = true }
+end
+
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local lfs = require("libs/libkoreader-lfs")
+local logger = require("logger")
+local _ = require("gettext")
+local T = require("ffi/util").template
+
+local CoverImage = WidgetContainer:new{
+    name = 'coverimage',
+    is_doc_only = true,
+}
+
+function CoverImage:_enabled()
+    return self.enabled
+end
+
+function CoverImage:_restore()
+    return self.restore
+end
+
+function CoverImage:init()
+    self.cover_image_path = G_reader_settings:readSetting("cover_image_path") or "Cover.png"
+    self.cover_restore_image_path = G_reader_settings:readSetting("cover_restore_image_path") or "Cover_restore.png"
```

Comment:
Name the setting for this one `cover_image_restore_path` so they all have the same prefix `cover_image_`.